dart/lib/src/http_client/breadcrumb_client.dart

```diff
@@ -58,8 +58,8 @@ class BreadcrumbClient extends BaseClient {
     String? reason;
     int? responseBodySize;
 
-    final stopwatch = Stopwatch();
-    stopwatch.start();
+    // final stopwatch = Stopwatch();
```

Comment:
should we drop such things since we have span support now?

Reply:
It's in the docs as expected feature for HTTP integrations.

Reply:
makes sense